src/components/Card.tsx

```diff
@@ -0,0 +1,356 @@
+import { ReactNode } from "react";
+import { breakpoints, colors } from "../theme";
+import { availablePoints, Step } from "../types";
+import styled from "styled-components";
+import cn from "classnames";
+
+export const InnerStepCard = styled.div`
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  flex: 1 1 auto;
+  min-height: 400px;
+  border-radius: 0.25rem;
+  margin: 0 auto 5rem auto;
+  border: 1px solid ${colors.palette.light};
+  border-radius: 0.25rem;
+  background-color: white;
+
+  ${breakpoints.desktop`
+    max-width: 1000px;
+    min-width: 750px;
+  `}
+`;
+
+export const OuterStepCard = styled.div`
+  padding: 2rem;
+
+  ${breakpoints.tablet`
+    padding: 0;
+  `}
+`;
+
+const LoadingCard = styled(InnerStepCard)`
+  min-width: 960px;
+  padding: 2rem;
+`;
+
+const StepCardHeader = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 16px 24px;
+  background: ${colors.card.background};
+  font-size: 1.8rem;
+  line-height: 3rem;
+
+  div {
+    display: flex;
+    align-items: center;
+  }
+
+  div.question-info {
+    font-weight: bold;
+
+    .ox-icon-lock {
+        margin-right: 1rem;
+    }
+  }
+
+  .num-questions, .points, .separator {
+      display: none;
+  }
+
+  .exercise-id, .separator {
+      font-weight: normal;
+  }
+
+  .separator {
+      margin: 0 1rem;
+  }
+
+  .exercise-id {
+      height: 28px; // Fix baseline issue
+  }
+
+  button {
+    color: ${colors.palette.gray};
+  }
+
+  .openstax-exercise-badges {
+      margin: 0;
+      line-height: 2rem;
+      svg {
+          display: block;
+          &:not(.interactive) {
+              margin: 0 0 0 6px !important;
+          }
+      }
+  }
+
+  ${breakpoints.desktop`
+      button.ox-icon-angle-left, button.ox-icon-angle-right {
+          display: none;
+      }
+      .separator {
+          display: inherit;
+      }
+  `}
+
+    /*
+    1. Show the arrows to move to previous and next question.
+    2. Show the number of questions.
+    3. Override box-shadow of icons when turned into a button.
+    */
```

Comment:
Though we aren't rendering these ourselves, I included the CSS in case Tutor ends up using this component. Could easily see moving this to Tutor though, especially if we don't use the same icons on the assignment service side.